controller/deis/settings.py

```diff
@@ -290,6 +290,9 @@
 # names which apps cannot reserve for routing
 DEIS_RESERVED_NAMES = ['deis']
 
+# the default expected status code for healthchecks
+DEIS_HEALTHCHECK_STATUS_CODE = '200'
```

Comment:
I made a mini comment about this in the commit:

> By default, the expected status code is 200 unless the user specifies
otherwise. This variable is non-configurable on the platform side as
both the controller and the publisher have to rely on this value, and I
could not find a happy medium for setting up a shared config variable.

Since it affects two components, I couldn't figure out a happy medium, so I left that part out :)